setup.py

```diff
@@ -17,6 +17,10 @@
     },
 }
 
+with open("requirements.txt", encoding="utf-8") as f:
```

Comment:
not-blocking: I'll suggest to use setup.cfg, which brings multiple advantages such reading files (requirements.txt, LICENSE) with simple `file: <path>`, requirements + extra (for litn, unit, func tests or building documentation) requirements could be define in single place.

Reply:
I suggest we hold off that because we will lose the ability to use scm versioning. And moving from `setup.py` to `setup.cfg` (both are legacy) may not be worthy.